### ISSUE TITLE
Windows version does not work with text to speech accessibility software.

### ISSUE THREAD

**Issue by RealAmethyst** [Xpl0itU/WiiUDownloader#114]
Hi there. I am a blind computer user who uses text to speech and only a keyboard to navigate my PC. This app does not speak options when I use my text to speech. I am also not sure if there is possible to navigate teh app only via keyboard, as I can't test due to no speech output. Could this please be looked into? To test this yourself, start the built in windows screen reader by holding down the windows key, ctrl and pressing enter, then try to navigate the UI using only keyboard.
